src/PreProcess/PreProcessKARC.jl

```diff
@@ -1,13 +1,16 @@
 function preprocess(PData::PDataKARC, Hop, g, calls, max_calls)
-    ζ = PData.ζ
+    ζ, ξ, maxtol, mintol = PData.ζ, PData.ξ, PData.maxtol, PData.mintol
     nshifts = PData.nshifts
     shifts = PData.shifts
 
     n = length(g)
     gNorm2 = norm(g)
-    precision = max(1e-12, min(0.5, (gNorm2^ζ)))
-    ϵ = 1e-12#sqrt(eps()) # * 100.0
-    cgtol = max(ϵ, min(0.09, 0.01 * norm(g)^(1.0 + ζ)))
+    # precision = max(1e-12, min(0.5, (gNorm2^ζ)))
+    # Tolerance used in Assumption 2.6b in the paper ( ξ > 0, 0 < ζ ≤ 1 )
+    cgtol = min(maxtol, ξ * gNorm2^(1.0 + ζ))
```

Comment:
```suggestion
    cgtol = min(maxtol, ξ * gNorm2^(1 + ζ))
```
Also: call it `cgatol`?